Application/src/view/visuals/component/Message.java

```diff
@@ -231,11 +237,26 @@ public void setStatic(boolean staticIndicator)  {
     public void renderDefault(GraphicsContext gc) {
 
 
+        if(trails.size() != 0) {
+            Trail firstIndex;
+            //Draws all trails except the first and last one
+            for (int i = 1; i < trails.size() - 1; i++) {
+                firstIndex = trails.get(i);
```

Comment:
I don't understand the name "firstIndex"?

Reply:
call it t  and leave it unchanged? I don't see any change from my code, but the variable name?

Reply:
gc.drawImage(trail, t.getXcoordinate(), (t.getYcoordinate() + 18), t.getWidth(), t.getHeight());

Reply:
@IsabelleTornqvist ok